src/components/PeopleTable.tsx

```diff
@@ -1,4 +1,39 @@
-export const PeopleTable = () => {
+import React from 'react';
+import { useSearchParams } from 'react-router-dom';
+import classNames from 'classnames';
+import { PersonType, SortType } from '../types/Person';
+import { Person } from './Person';
+import { SearchLink } from './SearchLink';
+
+type Props = {
+  people: PersonType[];
+  selectedPersonSlug: string;
+};
+
+export const PeopleTable: React.FC<Props> = ({
+  people,
+  selectedPersonSlug,
+}) => {
+  const [searchParams] = useSearchParams();
```

Comment:
Consider renaming `order` variable to avoid confusion with the built-in `order` function 🤔